### PR TITLE
Trying to increase coverage

### PR DESCRIPTION
**Pull request by csabadenes**

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -9,3 +9,18 @@ def test_region():
 def test_business_area():
     ab = BusinessArea(name="test_name")
     assert str(ab) == 'test_name'
+
+
+def test_user():
+    user = User(username="user_name")
```
Comment (domdinicola) [inline tests/test_models.py]: can we use a factory here?

Comment (csabadenes) [reply tests/test_models.py]: i have created an user factory for other places, but i don't see how it could help in this case

File: .gitignore [hunk 2]
```diff
@@ -57,3 +59,4 @@ coverage.xml
 Pipfile
 Pipfile.lock
 celerybeat.pid
+~build/
```
Comment (domdinicola) [inline .gitignore]: duplicate

File: src/unicef_security/graph.py [hunk 3]
```diff
@@ -23,7 +23,9 @@
 
 DJANGOUSERMAP = {'_pk': ['username'],
                  'username': 'userPrincipalName',
-                 'email': 'mail',
+                 # Looks like sometimes email is kept in `userPrincipalName` and `mail` is empty
```
Comment (domdinicola) [inline src/unicef_security/graph.py]: @saxix can you review this?

File: tox.ini [hunk 4]
```diff
@@ -1,5 +1,6 @@
 [tox]
 envlist = py{36,37}-d{21,22}
+# envlist = py{37}-d{22}
```
Comment (domdinicola) [inline tox.ini]: remove this

Comment (csabadenes) [reply tox.ini]: Done

File: src/unicef_security/graph.py (hunk 3 above)
Comment (csabadenes) [reply src/unicef_security/graph.py]: I don't know what's happening in Azure AD that my user has this field as `Null`, and i can't set it anywhere. I have checked other users too, and the `email` field seems to be kept in `userPrincipalName`. 
After/If this change is rolled back, the cassettes have to be regenerated with an Azure AD user that has the `mail` field set. After that most likely few of the test have to be adjusted to match the new data in the cassettes.

**Pull request merged**